src/models/identifier_count.ts

```diff
@@ -0,0 +1,26 @@
+import mongoose, { Schema, model } from "mongoose";
```

Comment:
wouldn't `counters` be a better name for this collection?
for reference:- https://www.mongodb.com/docs/v2.2/tutorial/create-an-auto-incrementing-field/#auto-increment-counters-collection
they're making use of `userId` to target counter that is used for user collection